app/views/email/_mailing_list_post.html.erb

```diff
@@ -1,5 +1,5 @@
 <li>
-  <a href='<%= Discourse.base_url + topic.relative_url %>' style='color: #<%= @anchor_color %>'><%= raw format_topic_title(topic.title) %></a>
+  <a href='<%= Discourse.base_url_no_prefix + topic.relative_url %>' style='color: #<%= @anchor_color %>'><%= raw format_topic_title(topic.title) %></a>
```

Comment:
If we set subfolder to "/forum" then `topic.relative_url` would be **/forum/t/-/123**